vmm/sandbox/src/cloud_hypervisor/config.rs

```diff
@@ -42,11 +43,28 @@ impl Default for CloudHypervisorVMConfig {
             common: Default::default(),
             hugepages: false,
             entropy_source: "/dev/urandom".to_string(),
+            task: Default::default(),
             virtiofsd: Default::default(),
         }
     }
 }
 
+#[derive(CmdLineParamSet, Deserialize, Clone, Serialize)]
+pub struct TaskConfig {
+    #[param(ignore)]
+    pub debug: bool,
+    pub sharefs_type: String,
```

Comment:
We do not have any support plan on `virtio-9p` because it's likely to be replaced by `virtiofs`.